src/wallet/walletdb.cpp

```diff
@@ -315,35 +351,35 @@ bool CWalletDB::WriteAutoCombineSettings(bool fEnable, CAmount nCombineThreshold
     std::pair<bool, CAmount> pSettings;
     pSettings.first = fEnable;
     pSettings.second = nCombineThreshold;
-    return batch.Write(std::string("autocombinesettings"), pSettings, true);
+    return batch.Write(std::string(DBKeys::AUTOCOMBINE), pSettings, true);
 }
 
 bool CWalletDB::ReadPool(int64_t nPool, CKeyPool& keypool)
 {
-    return batch.Read(std::make_pair(std::string("pool"), nPool), keypool);
+    return batch.Read(std::make_pair(std::string(DBKeys::POOL), nPool), keypool);
 }
 
 bool CWalletDB::WritePool(int64_t nPool, const CKeyPool& keypool)
 {
     nWalletDBUpdateCounter++;
-    return batch.Write(std::make_pair(std::string("pool"), nPool), keypool);
+    return batch.Write(std::make_pair(std::string(DBKeys::POOL), nPool), keypool);
 }
 
 bool CWalletDB::ErasePool(int64_t nPool)
 {
     nWalletDBUpdateCounter++;
-    return batch.Erase(std::make_pair(std::string("pool"), nPool));
+    return batch.Erase(std::make_pair(std::string(DBKeys::POOL), nPool));
 }
 
 bool CWalletDB::WriteMinVersion(int nVersion)
 {
-    return batch.Write(std::string("minversion"), nVersion);
+    return batch.Write(std::string(DBKeys::MINVERSION), nVersion);
 }
 
 bool CWalletDB::WriteHDChain(const CHDChain& chain)
 {
     nWalletDBUpdateCounter++;
-    std::string key = std::string("hdchain");
+    std::string key = std::string(DBKeys::HDCHAIN);
     if (chain.chainType == HDChain::ChainCounterType::Sapling)
         key += std::string("_sap");
```

Comment:
Nit: can use `SAP_HDCHAIN` here for consistency.
```diff
Index: src/wallet/walletdb.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/wallet/walletdb.cpp b/src/wallet/walletdb.cpp
--- a/src/wallet/walletdb.cpp	(revision ffddf66ac12b3cd6ee2cbc1ea77f46a6d1357785)
+++ b/src/wallet/walletdb.cpp	(date 1617929429351)
@@ -381,7 +381,7 @@
     nWalletDBUpdateCounter++;
     std::string key = std::string(DBKeys::HDCHAIN);
     if (chain.chainType == HDChain::ChainCounterType::Sapling)
-        key += std::string("_sap");
+        key = std::string(DBKeys::SAP_HDCHAIN);
     return batch.Write(key, chain);
 }
 
```

Reply:
Done, added.